README.md

```diff
@@ -54,19 +54,42 @@ cd bank-of-anthos
 
 3. **Create a GKE cluster.**
 
+- GKE Autopilot mode (see [GKE Autopilot overview](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview) to learn more):
+
+```
+gcloud services enable container.googleapis.com monitoring.googleapis.com \
+  --project ${PROJECT_ID}
+
+REGION=us-central1
+gcloud container clusters create-auto bank-of-anthos \
```

Comment:
Hi Tony, for consistency with the standard cluster below, could we make this a zonal (instead of regional) cluster? Thanks!

Reply:
Received clarification on this from @subfuzion: 

```
Autopilot automatically applies best practices for high availability in the region by creating three zones. To specify a single region requires configuring zonal affinity, and that would impact more of the demo than this minor PR does. William Denniss (Autopilot PM) provides much more detail here in his blog post: https://wdenniss.com/autopilot-single-zone
```

Merging as is, w/ regional autopilot cluster.